websites/I/India Today News/presence.ts

```diff
@@ -0,0 +1,541 @@
+const presence = new Presence({
+		clientId: "1051110545723506718",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+async function getStrings() {
+	return presence.getStrings(
+		{
+			play: "general.playing",
+			pause: "general.paused",
+		},
+		await presence.getSetting<string>("lang").catch(() => "en")
+	);
+}
+
+interface iFrameData {
+	duration: number;
+	currentTime: number;
+	paused: boolean;
+}
+
+let video: iFrameData = {
+	duration: 0,
+	currentTime: 0,
+	paused: true,
+};
+
+presence.on(
+	"iFrameData",
+	(data: { duration: number; currentTime: number; paused: boolean }) => {
+		video = data;
+	}
+);
+
+enum Assets {
+	Logo = "https://i.imgur.com/JJHnTn3.png",
+	Reading = "https://i.imgur.com/8vMPNni.png",
+	Searching = "https://i.imgur.com/oGQtnIY.png",
+	Pause = "https://i.imgur.com/0A75vqT.png",
+	Play = "https://i.imgur.com/Dj5dekr.png",
+	Live = "https://i.imgur.com/wtweZhK.png",
+}
+
+presence.on("UpdateData", async () => {
+	let presenceData: PresenceData = {
+			largeImageKey: Assets.Logo,
+			details: "Unsupported page",
+			smallImageKey: Assets.Logo,
```

Comment:
Why?

Reply:
The service is extremely large, so I did that in case a page is not supported by the presence.

Reply:
no i mean the small imagekey. why?